README.md

```diff
@@ -65,7 +65,7 @@ There are some guidelines which will make applying PRs easier for us:
  Optional supplemental description.
 ```
 + Make sure you have added the necessary tests (JUnit/IT) for your changes.
-+ Run all the tests with `mvn -Prun-its verify` to assure nothing else was accidentally broken.
++ Run all the tests with `mvn -Prun-its verify` to assure nothing else was accidentally broken (you can need to `mvn install` before).
```

Comment:
Hint: It should *never* need an `install`!

Reply:
@bmarwell well I will not enter into the debate but this is not my change but the way the project was setup, invoker+mrm config does not seem to work reliably without an install so added the comment.